workflows/Snakefile

```diff
@@ -13,33 +17,33 @@ params_md["fnr"] = 0.097
 params_md["na_rate"] = 0.014
 params_md["observe_homozygous"] = ["False", "True"]
 
+# MCMC parameters of mark00
+params_mcmc = dict()
+params_mcmc["seed"] = 42
+params_mcmc["n_trees"] = 1
+params_mcmc["n_cells"] = 2000
+params_mcmc["n_mutations"] = 50
+params_mcmc["strategy"] = """UNIFORM_INCLUDE_ROOT"""
+params_mcmc["fpr"] = 1.24*10**(-6)
+params_mcmc["fnr"] = 0.097
+params_mcmc["na_rate"] = 0.014
+params_mcmc["observe_homozygous"] = "False"
+
```

Comment:
params not in use yet - will come later once I define experiments - current snakemake rules can handle all possible parameters